cultcargo/sunblocker.yml

```diff
@@ -0,0 +1,148 @@
+_include:
+  - genesis/cult-cargo-base.yml
+
+cabs:
+    sunblocker:
+        info: A pyrap method to remove solar-like RFI from interferometric data (https://github.com/gigjozsa/sunblocker)
+        image:
+            _use: vars.cult-cargo.images
+            name: sunblocker
+        flavour:
+            kind: python-code
+            input_vars: false
+            input_dict: options
+        command: |
+            import subprocess
+            from sunblocker.sunblocker import Sunblocker
+            from omegaconf import OmegaConf
+           
+            #subprocess.call("pip install matplotlib -U")
+            func = options.pop('command')
+            run_func = getattr(Sunblocker(), func, None)
+            if run_func is None:
+                raise RuntimeError(f"Function {function} is not part of Sunblocker()") 
+            for arg in options.keys():
+                if options[arg] in [None]:
```

Comment:
Why not options[arg] is None, simpler? Or options = {key: value for key, value in options.items() if value is not None}, but I'm overly fond of list/dict comprehensions, I know. :)

Reply:
neat